settings.gradle.kts

```diff
@@ -14,9 +14,12 @@ includeBuild("build-logic")
 enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
 
 pluginManagement {
+  includeBuild("build-logic")
+
   repositories {
     mavenCentral()
     google()
+    gradlePluginPortal()
```

Comment:
Do you mind adding a content block?

```kotlin
      content {
        includeModule("foo", "bar")
      }
```

`gradlePluginPortal()` transparently redirects to other repos so I'd trust the other repos more than `gradlePluginPortal()`